frontend/src/components/fields/WavenumberRangeSlider.tsx

```diff
@@ -79,7 +79,7 @@ export const WavenumberRangeSlider: React.FC<WavelengthRangeSliderProps> = ({
   return (
     <div>
       <Typography id="input-slider" gutterBottom>
-        {isUnitChanged ? "Wavenumber length (nm)" : " Wavenumber range (cm-1)"}
+        {isUnitChanged ? " Wavelength (nm)" : " Wavenumber range (cm-1)"}
```

Comment:
```suggestion
        {isUnitChanged ? " Wavelength range (nm)" : " Wavenumber range (cm-1)"}
```
maybe for it to match?

I'd also maybe consider removing the units (nm, cm-1) from the label since we also see the units to the right.

Reply:
Ok to add "range", I think it's good to keep the units in the label (usual in scientific graphs)